package.json

```diff
@@ -4,17 +4,19 @@
   "version": "0.0.1",
   "scripts": {
     "start": "run-p keycloak start:jsonserver vite",
-    "start:dev": "run-p keycloak vite",    
+    "start:dev": "run-p keycloak vite",
     "keycloak": "cd keycloak && docker compose up",
     "prestart:jsonserver": "node mocks/createMockDb.js",
-    "start:jsonserver": "node mocks/apiServer.js",    
+    "start:jsonserver": "node mocks/apiServer.js",
     "test": "TZ=UTC jest --coverage",
     "monitor": "TZ=UTC jest --watchAll",
     "lint:fix": "eslint --ext .js,.jsx src --fix",
     "lint": "eslint --ext .js,.jsx src",
     "build": "vite build",
     "preview": "vite preview",
-    "vite": "vite"
+    "vite": "vite",
+    "localdev": "run-p localdev:init start:jsonserver vite ",
```

Comment:
should this be `run-p localdev jsonserver vite`
and we should be using pre/post prefixes for the init. Should we always start the jsonserver (that's probably a sensible choice). I think we should tidy up these scripts and drop out keycloak. The timezone env var is probably a good candidate for also going in the .env file

Reply:
Realised some other issue with the npm script already depending on run-p before npm install is guaranteed to have been called so moved this into the entry point as a shell command. This means there are no changes to the script section so the configuration/structure can be assessed elsewhere as this PR no longer touches them